src/Dockerfile

```diff
@@ -19,7 +25,25 @@ RUN yarn install --production --frozen-lockfile
 
 ADD . $app
 RUN bundle outdated --strict
-RUN mkdir -p tmp/pids
 RUN bundle exec rake assets:precompile
+RUN rm -rf node_modules tmp/* log/*
+
+
+FROM ruby:2.6.3-slim
+
+RUN apt update &&\
+    apt install -qq -y postgresql-client\
+    --fix-missing --no-install-recommends
+
+ENV app /app
+ENV INSTALL_PATH $app
+ENV RAILS_ENV production
+
+RUN mkdir -p tmp/pids
+
+WORKDIR $app
```

Comment:
this needs to be swapped as line 42 need to be executed inside `/app` and we didn't change the working directory until line 44.
That leads me to question the need for line 42 at all if these steps worked after build.

Reply:
I think it worked fine because the default working path in this image is set to `app`

Reply:
I mean the parent image default working path